src/binarySearch.js

```diff
@@ -0,0 +1,22 @@
+const BinarySearch = (inputArray, searchValue) => {
+  let max = inputArray.length - 1
+  let min = 0
+  while (min <= max) {
+    let middle = (min + max) / 2
+    if(inputArray[middle] === searchValue) {
+      return middle
+    }
+    else {
```

Comment:
I think this could just be an ```else if``` and then an ```else``` ?  You could use a search variable, like ```currentIndex```. This might make the code a little cleaner, but I don't know if it would run faster.